akvo/rsr/models/project.py

```diff
@@ -92,27 +92,54 @@ class Project(TimestampsMixin, models.Model):
     )
 
     STATUSES_COLORS = {
-        STATUS_NONE: 'black',
-        STATUS_NEEDS_FUNDING: 'orange',
-        STATUS_ACTIVE: '#AFF167',
-        STATUS_COMPLETE: 'grey',
-        STATUS_CANCELLED: 'red',
-        STATUS_ARCHIVED: 'grey',
+        '0': 'grey',
+        '1': 'orange',
+        '2': '#AFF167',
+        '3': 'grey',
+        '4': 'grey',
+        '5': 'red',
+        '6': 'grey',
     }
 
+    CODE_TO_STATUS = {
+        '0': 'N',
+        '1': 'H',
+        '2': 'A',
+        '3': 'C',
+        '4': 'C',
+        '5': 'L',
+        '6': 'R'
+    }
+
+    STATUS_TO_CODE = {
+        'N': '0',
+        'H': '1',
+        'A': '2',
+        'C': '3',
+        'L': '5',
+        'R': '6'
+    }
+
+    # Status combinations used in conditionals
+    EDIT_DISABLED = ['3', '5']
+    DONATE_DISABLED = ['0', '3', '4', '5', '6']
+    NOT_SUSPENDED = ['1', '2', '3', '4', '5']
+
     title = ValidXMLCharField(_(u'project title'), max_length=200, db_index=True, blank=True)
     subtitle = ValidXMLCharField(_(u'project subtitle'), max_length=200, blank=True)
-    status = ValidXMLCharField(
-        _(u'status'), max_length=1, choices=STATUSES, db_index=True, default=STATUS_NONE,
-        help_text=_(u'There are five different project statuses:<br/>'
-                    u'1) Needs funding: this project still needs funding and implementation has '
-                    u'not yet started.<br/>'
-                    u'2) Active: the implementation phase has begun.<br/>'
-                    u'3) Completed: the project has been completed.<br/>'
-                    u'4) Cancelled: the project never took place or work stopped before it was '
-                    u'fully implemented.<br/>'
-                    u'5) Archived: projects are archived when the reporting partner no longer uses '
-                    u'RSR.')
+    status = ValidXMLCharField(_(u'status'), max_length=1, choices=STATUSES, db_index=True, default=STATUS_NONE)
+    iati_status = ValidXMLCharField(
+        _(u'iati status'), max_length=1, choices=([('0', '')] + codelist_choices(ACTIVITY_STATUS)),
+        db_index=True, default='0',
+        help_text=_(u'There are six different project statuses:<br/>'
+                    u'1) Pipeline/identification: the project is being scoped or planned<br/>'
+                    u'2) Implementation: the project is currently being implemented<br/>'
+                    u'3) Completion: the project is complete or the final disbursement has been made<br/>'
+                    u'4) Post-completion: the project is complete or the final disbursement has been made, '
+                    u'but the activity remains open pending financial sign off or M&E<br/>'
```

Comment:
".. but the activity .." --> ".. but the project ..". Also change this in the migration.